Userland/Libraries/LibGfx/ImageFormats/ISOBMFF/JPEG2000Boxes.h

```diff
@@ -25,6 +25,43 @@ struct JPEG2000ImageHeaderBox final : public Box {
     u8 compression_type { 0 };
     u8 is_colorspace_unknown { 0 };
     u8 contains_intellectual_property_rights { 0 };
+
```

Comment:
I don't understand why you are adding these enums if we can't use them 🤔

Reply:
Also don't enums allow not listed values?

Reply:
We can use them to compare against, see 4th commit. But we "can't" use them as the type of the field. (Since it's just an enum, not an enum class, comparing ints against them works without casts.)

enums do allow not listed values, especially with an explicit underlying type, but that's something we don't usually use. I think clang doesn't warn if you switch over an enum and handle all listed values, and for gcc the kinda automatic fix for the diag is to add a `VERIFY_NOT_REACHED()` after the switch, which here would be wrong.